inst/project_templates/leej22_template_files/02_data_checks.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: "{{folder_name}}"
+subtitle: "Data Checks"
+author: "Jasme Lee and "
+date: "`r format(Sys.Date(), format = '%B %d, %Y')`"
+output:
+  html_document: 
+    fig_caption: yes
+    toc: true
+    toc_float: true
+    toc_collapsed: false
+    number_sections: true
+    toc_depth: 3
+    theme: flatly
+knit: (function(input, ...) {
+    rmarkdown::render(
+      input,
+      output_file =  here::here(glue::glue('reports/{{folder_name}}_02_data_checks_v01_{Sys.Date()}.html')),
```

Comment:
since you're using git, do you need to append the date to the file each time you run it?

Reply:
mainly its for when passing the html to PI. i like to have the version and date for referencing